src/pages/query-view/query-view.js

```diff
@@ -86,8 +86,13 @@ const cleanseRow = row => {
 const cleanseField = value => {
   if (typeof value === "boolean") {
     return value.toString();
+  } else if (typeof value === "object") {
```

Comment:
I think this will work for arrays too, but you may want to double check.

Reply:
It does! Thanks for thinking of that, didn't even occur to us to be afraid of arrays.